torch/masked/_ops.py

```diff
@@ -1486,6 +1485,48 @@ def logaddexp(
     input_mask: Optional[Tensor] = None,
     other_mask: Optional[Tensor] = None,
 ) -> Tensor:
+    """logaddexp(input, other, *, dtype=None, input_mask=None, other_mask=None) -> Tensor
+
+Returns logaddexp of all the elements in the :attr:`input` and the :attr:`other`
+tensor. The :attr:`input` elements are masked out according to the boolean tensor
+:attr:`input_mask` and the attr:`other` elements are masked out according to the boolean tensor
+:attr:`other_mask`. For the logaddexp operation, the value for masked out elements is ``-2147483648``.
```

Comment:
Is -2147483648 is true for logaddexp? (looks like it taken from logsumexp docstring).

Reply:
I'm sorry, but "For the logaddexp operation, the value for masked out elements is ``-2147483648``" doesn't really make sense. Could we please elaborate or remove this?

Either an element is masked out, which means it has a position, but no value, or it is not. A masked out element cannot have a value.

Reply:
> (looks like it taken from logsumexp docstring)

It is. 

The real answer for both logsumexp and logaddexp is the smallest value which the tensor's dtype can hold. `-torch.inf` for float types and `torch.iinfo.min` for integral.

https://github.com/pytorch/pytorch/blob/498a76080299d44a3e22787bc42ec8e2eba71905/torch/masked/_ops.py#L421-L425